analysis/combiners.py

```diff
@@ -644,6 +645,169 @@ def explain_computation(self):
         pass
 
 
+class SumAggregateErrorMetricsCombiner(pipeline_dp.Combiner):
+    """A combiner for aggregating errors across partitions for Sum"""
+    AccumulatorType = AggregateErrorMetricsAccumulator
+
+    def __init__(self, error_quantiles: List[float]):
+        # The contribution bounding error is negative, so quantiles <0.5 for the
+        # error distribution (which is the sum of the noise and the contribution
+        # bounding error) should be used to come up with the worst error
+        # quantiles.
+        self._error_quantiles = [(1 - q) for q in error_quantiles]
+
+    def create_accumulator(self,
+                           metrics: metrics.SumMetrics,
+                           probability_to_keep: float = 1) -> AccumulatorType:
+        """Creates an accumulator for metrics."""
+        # Data drop ratio metrics
+        total_aggregate = metrics.sum
+        # TODO: Compute data_dropped metrics
+        data_dropped_l0 = 0
+        data_dropped_linf = 0
+        data_dropped_partition_selection = 0
+
+        # Absolute error metrics
+        abs_error_l0_expected = probability_to_keep * metrics.expected_cross_partition_error
+        abs_error_linf_expected = probability_to_keep * (
+            metrics.per_partition_error_min + metrics.per_partition_error_max)
+        abs_error_l0_variance = probability_to_keep * metrics.std_cross_partition_error**2
+        abs_error_variance = probability_to_keep * (
+            metrics.std_cross_partition_error**2 + metrics.std_noise**2)
+        loc_cpe_ne = metrics.expected_cross_partition_error
+        std_cpe_ne = math.sqrt(metrics.std_cross_partition_error**2 +
+                               metrics.std_noise**2)
+        abs_error_quantiles = []
```

Comment:
Computing quantiles is complicated, could you please extract it in a separate method/function (and it would be great to unify that with count computations)